pkgs/by-name/kc/kcl-cli/package.nix

```
@@ -0,0 +1,32 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
```

```suggestion
buildGoModule (finalAttrs: {
```
This makes this overrideable